integrations.json

```diff
@@ -84,13 +84,13 @@
           "assembly": "System.Data",
           "type": "System.Data.SqlClient.SqlCommand",
           "method": "ExecuteReader",
-          "signature": [ 32, 2, 12, 82, 8, 11, 82, 91, 14 ]
+          "signature": [ 32, 1, 12, 87, 12, 11, 92 ]
```

Comment:
This changes the target from `ExecuteReader(CommandBehavior,string)` to `ExecuteReader(CommandBehavior)`, because the former doesn't exist in newer versions of `SqlCommand`.